src/views/TimersView.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import Stopwatch from '../components/timers/Stopwatch';
+import Countdown from '../components/timers/Countdown';
+import XY from '../components/timers/XY';
+import Tabata from '../components/timers/Tabata';
+import './TimersView.css';
+
+const TimersView = () => {
```

Comment:
This view should have a list of timers that the user has added to the workout from the `/add` path